src/coreclr/src/tools/aot/ILCompiler.ReadyToRun/JitInterface/CorInfoImpl.ReadyToRun.cs

```diff
@@ -1021,28 +1025,9 @@ private void getFieldInfo(ref CORINFO_RESOLVED_TOKEN pResolvedToken, CORINFO_MET
                     }
                     else
                     {
-                        if ((flags & CORINFO_ACCESS_FLAGS.CORINFO_ACCESS_ADDRESS) != 0)
```

Comment:
As far as I can tell, this case is handled just fine now. This makes a bunch of exceptions thrown during corelib compilation to go away.

Reply:
Please don't fix this issue in this PR. I believe this needs to be implemented and tested independently. We should add a test to crossgen2smoke to verify that this is ok. (I'll put this together today.)

---
In reply to: [445983213](https://github.com/dotnet/runtime/pull/38229#discussion_r445983213) [](ancestors = 445983213)

Reply:
I believe it is a low-risk change to remove this:
- This check prohibits `ldsflda` IL instruction. However, `ldsfld`/`stsfld` instructions are allowed and handled fine.
- There is no a check like this in the old crossgen.
- We have coverage for `ldsflda` via many existing tests.